module/PowerNSX.psm1

```diff
@@ -10004,6 +10005,95 @@ function New-NsxLogicalSwitch  {
     end {}
 }
 
+
+function Set-NsxLogicalSwitch {
+
+    <#
+    .SYNOPSIS
+    Configures an existing NSX Logical Switch.
+
+    .DESCRIPTION
+    An NSX Logical Switch provides L2 connectivity to VMs attached to it.
+    A Logical Switch is 'bound' to a Transport Zone, and only hosts that are
+    members of the Transport Zone are able to host VMs connected to a Logical
+    Switch that is bound to it.  All Logical Switch operations require a
+    Transport Zone.
+
+    .EXAMPLE
+    Change the name on an existing Logical Switch (LS1 to LS2)
+
+    PS C:\> $ls = Get-NsxLogicalSwitch -name LS1 | Set-NsxLogicalSwitch -name LS2
```

Comment:
set command before description

add also more example ? (for tenant, description...)